homeassistant/components/amcrest/sensor.py

```diff
@@ -33,7 +41,7 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
     device = hass.data[DATA_AMCREST][DEVICES][name]
     async_add_entities(
         [
-            AmcrestSensor(name, device, sensor_type)
+            AmcrestSensor(name, device, SENSORS[sensor_type])
             for sensor_type in discovery_info[CONF_SENSORS]
         ],
```

Comment:
Iterate over the available descriptions instead.

Reply:
That does not sound correct, what happens if there are 2 sensors that fit the same description key.